api/src/main/java/org/apache/iceberg/exceptions/ValidationException.java

```diff
@@ -24,7 +24,11 @@
 import org.apache.iceberg.Schema;
 
 /**
- * Exception raised when validation checks fail.
+ * Exception which is raised when the arguments are valid in isolation,
```

Comment:
Small nit, can you try to make each line be longer and try to line up towards the right?  

I don't see many places where the doc line-breaks on a comma, usually it's on a period.  You can also add a <p> after the period.

A good example in my opinion : https://github.com/apache/iceberg/blob/master/api/src/main/java/org/apache/iceberg/ExpireSnapshots.java

Thank you

Reply:
ok, I‘m sorry for it.
I just want to make a part of sentence is complete in one line.